datajoint/schema.py

```diff
@@ -96,6 +98,46 @@ def size_on_disk(self):
             FROM information_schema.tables WHERE table_schema='{db}'
             """.format(db=self.database)).fetchone()[0])
 
+    def make_module_code(self):
```

Comment:
What's this for? Addition of an entirely new feature like this should not be done as part of a patch increment. I'd much rather you add this to the next release.